notebooks/jax-to-openvino/jax-classification-to-openvino.ipynb

```diff
@@ -99,7 +99,10 @@
    "outputs": [],
    "source": [
     "%pip install -q \"openvino>=2024.5.0\"\n",
```

Comment:
probably need to update openvino on nightly to resolve unsupported op issue (or some specific jax version is required)